From leitao@linux.vnet.ibm.com  Thu Oct  8 15:41:38 2009
From: Breno Leitão <leitao@linux.vnet.ibm.com>
Date: Thu, 24 Sep 2009 16:58:23 -0300
Subject: jsm: Rewriting a bad log message
To: greg@kroah.com
Cc: linux-kernel@vger.kernel.org, scottk@digi.com
Message-ID: <b37337c10bf763c5d64071301b45a2089e97897f.1253820690.git.leitao@linux.vnet.ibm.com>


Actually jsm displays "Device Added" 8 times (for a 8 port device).
This silly patch just makes things more informative, showing
the port (instead of the device) that was added.

Signed-off-by: Breno Leitão <leitao@linux.vnet.ibm.com>
Cc: Scott Kilau <scottk@digi.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/jsm/jsm_tty.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/serial/jsm/jsm_tty.c
+++ b/drivers/serial/jsm/jsm_tty.c
@@ -472,7 +472,7 @@ int __devinit jsm_uart_port_init(struct 
 		if (uart_add_one_port (&jsm_uart_driver, &brd->channels[i]->uart_port))
 			printk(KERN_INFO "jsm: add device failed\n");
 		else
-			printk(KERN_INFO "Added device \n");
+			printk(KERN_INFO "jsm: Port %d added\n", i);
 	}
 
 	jsm_printk(INIT, INFO, &brd->pci_dev, "finish\n");
